### ISSUE TITLE
Some issues about weighted graph

### ISSUE THREAD

**Issue by Yoyo-cool99** [jhljx/CTGCN#8]
Hi, your work is very meaningful. 
But I have some problems, I am performing a dynamic graph embedding algorithm(DynGEM) on my own undirected weighted graph dataset and the loss function cannot converge.
So, are these dynamic graph embedding algorithms suitable for undirected weighted graph datasets?
Thanks in advance for your answer!

![image](https://user-images.githubusercontent.com/50907124/186805971-e24192e8-e1b2-4b77-824a-a5f0614ef0b7.png)

**Comment by jhljx** [comment]
These dynamic graph embedding algorithms are suitable for undirected weighted graph. You could tune the learning rate or enlarge the batch size to see whether the loss function curve will converge or not.
